pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 dependencies = [
     "h5py>=3.6.0",
     "xarray>=0.20.2",
-    "pynxtools>=0.9.0",
+    "pynxtools @ git+https://github.com/FAIRmat-NFDI/pynxtools.git@master",
```

Comment:
installing this will require master branch from pynxtools (which is in conflict with some NOMAD deployments.

Reply:
I'm not sure I understand. Shoudn't the latest release be fine (which still needs updating the minimal requirements here)?

Reply:
I see now, I fixed it.